src/main/java/seedu/storemando/logic/commands/ClearCommand.java

```diff
@@ -17,7 +19,8 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "All items in the specified location (if any) are cleared!";
+    public static final String CLEAR_MESSAGE_SUCCESS = "All items in the list are cleared!";
```

Comment:
Should we change to "All items in the inventory are cleared" or something like that since clear deletes all items in the inventory and not the sublist?

Reply:
Will update!